test/deploy/kubernetes/solution-config/config.yaml

```diff
@@ -64,11 +64,41 @@ cortx:
     transport_type: libfab
     ios:
       endpoints:
-      - motr://data1-node1.cortx-cluster.com:3232
+      - inet:tcp:data-node1@28002
```

Comment:
This is totally different format... We need approval from Ujjwal for these?

Reply:
This is the new motr address format and these endpoints are for motr reference only.
Motr clients will refer the endpoints in this format only.